javascript-carousel/main.js

```diff
@@ -0,0 +1,196 @@
+    icon5.classList.replace('fa-regular', 'fa-solid');
+  }
+  if (imageCount > 5) {
+    image.setAttribute('src', 'images/025.png');
+    icon5.classList.replace('fa-solid', 'fa-regular');
+    icon1.classList.replace('fa-regular', 'fa-solid');
+    imageCount = 1;
+  }
+}
+
+var rightArrow = document.querySelector('.right-arrow');
+rightArrow.addEventListener('click', goToTheNextImageButton);
+
+function goToTheNextImageButton(event) {
+  imageCount++;
+
+  if (event.target && event.target.matches('.right-arrow')) {
+    clearInterval(intervalId);
+    intervalId = setInterval(goThruImages, 3000);
+  }
+  if (imageCount === 2) {
+    image.setAttribute('src', 'images/001.png');
+    icon1.classList.replace('fa-solid', 'fa-regular');
+    icon2.classList.replace('fa-regular', 'fa-solid');
+  }
+  if (imageCount === 3) {
+    image.setAttribute('src', 'images/004.png');
+    icon2.classList.replace('fa-solid', 'fa-regular');
+    icon3.classList.replace('fa-regular', 'fa-solid');
+  }
+  if (imageCount === 4) {
+    image.setAttribute('src', 'images/007.png');
+    icon3.classList.replace('fa-solid', 'fa-regular');
+    icon4.classList.replace('fa-regular', 'fa-solid');
+  }
+  if (imageCount === 5) {
+    image.setAttribute('src', 'images/039.png');
+    icon4.classList.replace('fa-solid', 'fa-regular');
+    icon5.classList.replace('fa-regular', 'fa-solid');
+  }
+  if (imageCount > 5) {
+    image.setAttribute('src', 'images/025.png');
+    icon5.classList.replace('fa-solid', 'fa-regular');
+    icon1.classList.replace('fa-regular', 'fa-solid');
+    imageCount = 1;
+  }
+}
+
+var leftArrow = document.querySelector('.left-arrow');
+leftArrow.addEventListener('click', goToPreviousImage);
+
+function goToPreviousImage(event) {
+  imageCount--;
+  if (event.target && event.target.matches('.left-arrow')) {
+    clearInterval(intervalId);
+    intervalId = setInterval(goThruImages, 3000);
+  }
+  if (imageCount === 0) {
+    image.setAttribute('src', 'images/039.png');
+    icon5.classList.replace('fa-regular', 'fa-solid');
+    icon1.classList.replace('fa-solid', 'fa-regular');
+    icon4.classList.replace('fa-solid', 'fa-regular');
+    imageCount = 5;
+  }
+  if (imageCount === 1) {
+    image.setAttribute('src', 'images/025.png');
+    icon1.classList.replace('fa-regular', 'fa-solid');
+    icon2.classList.replace('fa-solid', 'fa-regular');
+  }
+  if (imageCount === 2) {
+    image.setAttribute('src', 'images/001.png');
+    icon2.classList.replace('fa-regular', 'fa-solid');
+    icon3.classList.replace('fa-solid', 'fa-regular');
+  }
+  if (imageCount === 3) {
+    image.setAttribute('src', 'images/004.png');
+    icon3.classList.replace('fa-regular', 'fa-solid');
+    icon4.classList.replace('fa-solid', 'fa-regular');
+  }
+  if (imageCount === 4) {
+    image.setAttribute('src', 'images/007.png');
+    icon4.classList.replace('fa-regular', 'fa-solid');
+    icon5.classList.replace('fa-solid', 'fa-regular');
+  }
```

Comment:
You're using the `imageCount` variable here a lot like indexes of an array. If you make the list of image URLs into an array, you might be able to simplify this code by quite a lot.